Geoms/CMS-2017.cc

```diff
@@ -107,6 +107,62 @@ namespace
   }
 */
 
+  void SetupBackwardSearch_Iter1(IterationConfig& ic)
```

Comment:
the added methods look the same.
Can the copy-paste be avoided? e.g. `SetupBackwardSearch_PixelCommon`, which can be called by 1,3,5,6,9

Reply:
Done.

Reply:
Done.